openml-lightgbm/lightgbm-provider/src/test/java/com/feedzai/openml/provider/lightgbm/LightGBMModelCreatorTest.java

```diff
@@ -284,6 +285,23 @@ public void loadModelWithWrongNumberOfFieldsThrowsTest() {
         ).withMessage(ERROR_MSG_SCHEMA_WITH_WRONG_PREDICTIVE_FIELDS_SIZE);
     }
 
+    /**
+     * Assert that loading a model with a wrong field
+     * names order throws a ModelLoadingException.
+     *
+     * @since 1.0.18
+     */
+    @Test
+    public void loadModelWithWrongFieldNamesOrderThrowsTest() {
```

Comment:
Should we have a test also for a good schema, where it should work?

Reply:
The other tests already cover that, right?